src/tests/ftest/util/server_utils.py

```diff
@@ -30,14 +30,16 @@
 import subprocess
 import json
 import re
-import resource
 import signal
-import fcntl
 import errno
 import yaml
+import getpass
 
-from avocado.utils import genio
-from general_utils import pcmd
+from general_utils import pcmd, check_file_exists, poll_pattern
+from command_utils import JobManager, DaosCommand, Orterun
```

Comment:
(pylint-no-name-in-module) No name 'JobManager' in module 'command_utils'
(pylint-no-name-in-module) No name 'DaosCommand' in module 'command_utils'
(pylint-no-name-in-module) No name 'Orterun' in module 'command_utils'
(pylint-unused-import) Unused JobManager imported from command_utils